airbyte-server/src/test/java/io/airbyte/server/handlers/WorkspacesHandlerTest.java

```diff
@@ -69,7 +89,66 @@ private StandardWorkspace generateWorkspace() {
         .withDisplaySetupWizard(true)
         .withNews(false)
         .withAnonymousDataCollection(false)
-        .withSecurityUpdates(false);
+        .withSecurityUpdates(false)
+        .withTombstone(false);
+  }
+
+  @Test
+  void testCreateWorkspace() throws JsonValidationException, ConfigNotFoundException, IOException {
+    when(configRepository.listStandardWorkspaces())
+      .thenReturn(Collections.singletonList(workspace));
+
+    configRepository.writeStandardWorkspace(workspace);
+
+    final WorkspaceCreate workspaceCreate = new WorkspaceCreate()
+      .name("new workspace")
+      .initialSetupComplete(false)
+      .displaySetupWizard(true)
+      .news(false)
+      .anonymousDataCollection(false)
+      .securityUpdates(false);
+
+    final WorkspaceRead created = workspacesHandler.createWorkspace(workspaceCreate);
+
+    assertTrue(created.getWorkspaceId() instanceof UUID);
```

Comment:
The test is a bit brittle here. What about: Injecting the uuid generator in a protected constructor and mock it so you can control the value of the uuid and you can just do a `assertEqual(expectedWorkspaceRead, created)`

Here is an example of how we do it: `io/airbyte/server/handlers/DestinationDefinitionsHandler.java:53`